examples/nixnet/can-signal-single-point-output.py

```diff
@@ -87,9 +87,10 @@ def check_for_error(status):
     check_for_error(create_session_response.status)
```

Comment:
Unused variable `SUCCESS` on line 52 above. GitHub doesn't let me add comment there, so adding it here.

Reply:
Done